js/index.js

```diff
@@ -36,34 +36,88 @@ function renderPepperoni() {
   document.querySelectorAll('.pep').forEach(onePep => {
     if (state.pepperoni) {
       onePep.style.visibility = 'visible';
+      
     } else {
       onePep.style.visibility = 'hidden';
     }
   });
 }
 
 function renderMushrooms() {
-  // Iteration 1: set the visibility of `<section class="mushroom">`
+  document.querySelectorAll('.mushroom').forEach(oneMush => {
+    if (state.mushroom) {
+      oneMush.style.visibility = 'visible';
+    } else {
+      oneMush.style.visibility = 'hidden';
+    }
+  });
 }
 
 function renderGreenPeppers() {
   // Iteration 1: set the visibility of `<section class="green-pepper">`
+  document.querySelectorAll('.green-pepper').forEach(onePepe => {
+    if (state.greenPeppers) {
+      onePepe.style.visibility = 'visible';
+    } else {
+      onePepe.style.visibility = 'hidden';
+    }
+  });
 }
 
 function renderWhiteSauce() {
   // Iteration 2: add/remove the class "sauce-white" of `<section class="sauce">`
+  document.querySelectorAll('.sauce').forEach(oneSauce => {
+    if (state.whiteSauce) {
+      oneSauce.style.visibility = 'visible';
+    } else {
+      oneSauce.style.visibility = 'hidden';
+    }
+  });
 }
 
 function renderGlutenFreeCrust() {
   // Iteration 2: add/remove the class "crust-gluten-free" of `<section class="crust">`
+  document.querySelectorAll('.crust').forEach(oneCrust => {
+    if (state.glutenFreeCrust) {
+      oneCrust.style.visibility = 'visible';
+    } else {
+      oneCrust.style.visibility = 'hidden';
+    }
+  });
 }
 
+
 function renderButtons() {
   // Iteration 3: add/remove the class "active" of each `<button class="btn">`
+  var btns = document.querySelectorAll(".btn");
```

Comment:
here you could have use add and remove.
and if you want to see best practices for this specific iteration let me give you andre´s approach :)
So you create an object to link state to the buttons, and then you loop ver the object and check which buttons needs to be toggled
![imagen](https://user-images.githubusercontent.com/19295928/78008037-69992700-733f-11ea-92e0-af403d53621d.png)